frontend/app/src/components/staking/liquity/LiquityStakingDetails.vue

```diff
@@ -51,29 +54,32 @@ import { Blockchain } from '@rotki/common/lib/blockchain';
 import {
   LiquityPoolDetail,
   LiquityPoolDetails,
-  LiquityStaking,
-  LiquityStakingEvent,
-  LiquityStakingEvents
+  LiquityStaking
 } from '@rotki/common/lib/liquity';
 import { ComputedRef } from 'vue';
 import RefreshButton from '@/components/helper/RefreshButton.vue';
+import TransactionContent from '@/components/history/transactions/TransactionContent.vue';
 import LiquityPools from '@/components/staking/liquity/LiquityPools.vue';
 import LiquityStake from '@/components/staking/liquity/LiquityStake.vue';
 import { isSectionLoading } from '@/composables/common';
-import { LiquityStakeEvents } from '@/premium/premium';
 import { useLiquityStore } from '@/store/defi/liquity';
+import { useTransactions } from '@/store/history/transactions';
+import { useTasks } from '@/store/tasks';
 import { Section } from '@/types/status';
+import { TaskType } from '@/types/task-type';
+import {
+  HistoryEventType,
+  TransactionEventProtocol
+} from '@/types/transaction';
 import { balanceSum } from '@/utils/calculation';
 import { uniqueStrings } from '@/utils/data';
 
 const selectedAccount = ref<GeneralAccount | null>(null);
-
 const liquityStore = useLiquityStore();
-const { staking, stakingEvents, stakingPools } = toRefs(liquityStore);
-const { fetchStaking, fetchStakingEvents, fetchPools } = liquityStore;
+const { staking, stakingPools } = toRefs(liquityStore);
+const { fetchStaking, fetchPools } = liquityStore;
```

Comment:
I don't see the liquity store in the change list, shouldn't we also remove the implementation details from it?

`stakingEvents` and `fetchStakingEvents` should not be used anywhere after this, correct?